changelogs/fragments/gha_cleanup_version_update.yaml

```
@@ -0,0 +1,6 @@
+---
+major_changes:
+- "Remove support for ansible-core 2.13"
```

```suggestion
breaking_changes:
- "Remove support for ansible-core < 2.14"
```

According to [Creating a changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment):

> Major changes mean the user can CHOOSE to make a change when they update but do not have to

But users don't have a choice. We're removing support for outdated ansible (core) versions. The only choice they have is to not use 3.0.0. But that's not a problem, breaking changes in a major release are OK ;-)

Additionally, we're not removing support for ansible-core 2.13 only but any version < 2.14. Let's make this clear.